### DISCUSSION TITLE
This view for all Feature

### DISCUSSION THREAD

**Discussion by Mannshoch** [j8sr0230/fc_nodes#133] [Q&A]
I normally Work with other CADs than FreeCAD. I found this Addon. 
Your nodes seems working similar like a Feature manager in CAD Systems like Inventor and Solidworks Work or like FreeeCAD may could work after solving the Toponaming problem. ( I like this Node Design) 

Classic feature Managers normally has not such a nice overview about their dependency.
 Do you see a way to allow show all Features in feature Tree in this view?

**Comment by j8sr0230** [comment]
In fact, I'm working on this improvement right now. that's why it's been a bit quiet in this repo for the last three weeks. In the future, all nodes will be displayed with all their properties in a tree view. The properties can then be manipulated either via the respective node widgets or in the tree view. This revision should take the node editor to a new level...
